wp-content/themes/reconasia/assets/_scss/style.scss

```diff
@@ -13,3 +13,4 @@
 @use 'components/share';
 @use 'layout/header';
 @use 'layout/footer';
+@use 'modules/print';
```

Comment:
We don't want to include the print styles as a part of our global styles. The idea is by separating them out into their own stylesheet, which is separately loaded in via `functions.php` is that these styles will only ever be loaded in if the user is printing the site - we don't want to also load them just during regular usage of the site.

Reply:
Thats makes sense! thanks for the clarification!